### PR TITLE
Float staff buttons to stop them from disappearing

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd weird little CSS fix

I want to keep this in platform because this should go upstream as well...

### CODE REVIEW

**Comment by stvstnfrd** [comment]
Screenshot?

**Comment by caesar2164** [comment]
Without `float: right;`:
screen shot 2018-05-18 at 3 17 40 pm


With `float: right;`:
screen shot 2018-05-18 at 3 17 25 pm

File: lms/static/sass/course/courseware/_courseware.scss [hunk 1]
```diff
@@ -19,6 +19,7 @@ html.video-fullscreen {
   @extend %ui-print-excluded;
 
   margin: ($baseline/2) ($baseline/4) 0 0;
+  float: right;
```
Comment (stvstnfrd) [inline lms/static/sass/course/courseware/_courseware.scss:22]: Why change here in platform and not on the theme?

Comment (stvstnfrd) [reply lms/static/sass/course/courseware/_courseware.scss:22]: Oh, I see your message about why...
Have you submitted this upstream yet?

**Review by stvstnfrd — APPROVED** [review]
I'm +1. I created a Trello card to submit this upstream too.

**Comment by stvstnfrd** [comment]
@caesar2164 Oh, and I'll pull this in to my branch, but leave this PR open here for now.

**Comment by caesar2164** [comment]
Hmmm, @stvstnfrd I just found this:

https://github.com/edx/edx-platform/pull/15693

**Comment by caesar2164** [comment]
I think we should remove my change and just cherry-pick that...

**Comment by stvstnfrd** [comment]
@caesar2164 can you test it locally and let me know?

**Comment by caesar2164** [comment]
We should grab this instead: https://github.com/edx/edx-platform/pull/15693.

**Comment by caesar2164** [comment]
worked great locally!

**Comment by stvstnfrd** [comment]
@caesar2164 Looks like you still need to push the cherry-pick?

**Pull request closed without merging**